site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1168,13 +1168,16 @@ public function getActiveUserIds(bool $instructor, bool $fullAccess, bool $limit
             $extra_join = ' INNER JOIN users ON courses_users.user_id = users.user_id ';
             $extra_where = ' OR users.user_access_level <= 2';
         }
+        $terms = ($term === '' ? 'courses.term' : $term);
+        $courses = ($course === '' ? 'courses.course' : $course);
         $result_rows = [];
         $this->submitty_db->query(
             "SELECT DISTINCT courses_users.user_id as user_id
                 FROM courses_users INNER JOIN courses
-                ON courses_users.term = courses.term AND courses_users.course = courses.course
+                ON courses_users.term = ? AND courses_users.course = ? 
                 " . $extra_join . "
-                WHERE courses.status = 1 AND user_group IN (" . implode(', ', $args) . ")" . $extra_where
+                WHERE courses.status = 1 AND user_group IN (" . implode(', ', $args) . ")" . $extra_where,
```

Comment:
It would be better to use `createParameterList()` here to prevent potential SQL injections if the args ever become user controlled in the future.